ff/src/fields/macros.rs

```diff
@@ -443,6 +443,31 @@ macro_rules! impl_Fp {
                 }
             }
 
```

Comment:
Can this just be a default method on the PrimeField trait? It seems to use only generic methods, and doesn’t seem to require any specific information about the PrimeField impl